### ISSUE TITLE
PWM[1].on_time_0 only works for 1 millisecond

### ISSUE THREAD

**Issue by Rocketmagnet** [shadow-robot/sr-ronex#27]
Using rostopic, set pwm/1 to Period=100, pwm_on_time_0=50, pwm_on_time_1=20.

Expected behaviour: PWM outputs appear on pins D2 and D3. Pwm outputs stay active indefinitely.

What actually happens: PWM outputs appear as expected on both pins, but the output on D2 stops after exactly 1ms.

This bug has been traced as far as the SPI connection to the ET1200 chip. The correct on time can be seen in one frame, but is then zero in the next frame. This means that the bug is almost certainly in the host driver.

Here we can see the PWM signal start then stop after the next frame.
![frames_both](https://f.cloud.github.com/assets/3330595/871911/4f9936e8-f857-11e2-9caf-3e294016aeaf.png)

Here we zoom in on the data from the ET1200 chip. We can see the values 100, 50, 20 being read as expeted.
![frame_0](https://f.cloud.github.com/assets/3330595/871912/55c3dff0-f857-11e2-8f53-cfa29f04b108.png)

The data in the next frame show that the value 50 has turned to 0, while the other values are still correct.
![frame_1](https://f.cloud.github.com/assets/3330595/871914/58dcb91e-f857-11e2-814b-10cbc5b8c757.png)

**Comment by toliver** [comment]
The reason for this is that we are mapping two things on the same pin.

in the file sr_ronex_mechanism_model/example/ronex.urdf we can find the following section:

``` xml
<transmission type="sr_ronex_mechanism_model/RonexTransmission" name="my_ronex_transmission">
    <joint name="joint1"/>
    <mapping ronex="ronex_33554433_0" property="position" analogue_pin="0" scale="1.0" offset="0.0"/>
    <mapping ronex="ronex_33554433_0" property="effort"   analogue_pin="1" scale="2.0" offset="1.0"/>
    <mapping ronex="ronex_33554433_0" property="command" pwm_module="1" pwm_pin="0"/>
</transmission>
```

The last mapping is mapping the effort command for the joint1 (this is an example where we define a robot with joints) to the pin 0 of the PWM module 1. As nobody is assigning a value to such an effort command, it's value is always 0. 

So basically, the passthrough controller sets a value (commanded by the user via topic) and in the next loop (hence the 1ms) ronex_transmission overwrites that value with a 0.

The solution is to remove the conflicting mapping line from the urdf.

**Comment by Rocketmagnet** [comment]
Tested and seen to work now.
